.changeset/rude-phones-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/core': minor
```

Comment:
We should add `wagmi` to this changeset too.

Reply:
done. Also snuck in some tests.